pkg/cli/list_diff.go

```diff
@@ -0,0 +1,66 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/semaphoreci/spc/pkg/consolelogger"
+	git "github.com/semaphoreci/spc/pkg/git"
+	logs "github.com/semaphoreci/spc/pkg/logs"
+	"github.com/spf13/cobra"
+)
+
+var listDiffCmd = &cobra.Command{
+	Use: "list-diff",
+
+	Run: func(cmd *cobra.Command, args []string) {
+		consolelogger.Enabled = false
+
+		defaultBranch := fetchOptionalStringFlag(cmd, "default-branch")
+		defaultRange := fetchOptionalStringFlag(cmd, "default-range")
+		branchRange := fetchOptionalStringFlag(cmd, "branch-range")
+		onTags := fetchOptionalBoolFlag(cmd, "on-tags")
+
+		gitSettings := git.NewDiffSet(defaultBranch, defaultRange, branchRange, onTags)
+
+		if gitSettings.IsEvaluationNeeded() {
+			fmt.Println("Running on a tag, skipping evaluation.")
```

Comment:
This will not be clear enough to customers who are seeing this for the first time. Either make it more clear, or remove it.